src/pyhf/compat.py

```diff
@@ -0,0 +1,76 @@
+"""
+Compatibility functions for translating between ROOT and pyhf
+"""
+
```

Comment:
Can you add a `__all__` and a `__dir__` function to this module and also add it to the `tests/test_public_api_repr.py` tests? c.f. PR #1403 for examples of what I mean.

Reply:
done